README.md

```diff
@@ -6,7 +6,7 @@ A Minecraft server implementation written in Python
 McPy aims to be as fast as possible, while keeping as many game mechanics same as vanilla, and including a plugin system by default.
 
 ## Where can I download McPy?
-McPy is currently under heavy development: very few things are actually working. However, there is a very early alpha avalible: all you need to do is download `startup.py` and run it with **PYTHON 3.8 OR HIGHER**. It'll exit the first time through: simply start it back up to run it. Press Ctrl+C to stop it.
+McPy is currently under heavy development: very few things are actually working. However, there is a very early alpha avalible: all you need to do is either download the .zip files from GitHub or clone the repository. Then run dependencies.py with **PYTHON 3.8 OR HIGHER** and wait for the dependencies to install and the server to start. Press Ctrl+C to stop it. Run main.py to start the server again.
```

Comment:
> either download the .zip files from GitHub or clone the repository

Both are basically cloning
> [...] . Press Ctrl+C to stop it. [...]


This sentence is to short in my opinion. Since this applies pretty much anywhere it could be rephrased into
"As with all terminal-based applications, you can use Ctrl + C to abort this task."

Reply:
Fixed by adding recommended lines.